packages/plugin-plot/src/plot-utils.ts

```diff
@@ -1,4 +1,13 @@
-import * as math from 'mathjs'
+import * as mathjs from 'mathjs'
+
+let math = mathjs
+
+export function createInstance(imported: math.ImportObject) {
+  math = mathjs.create(mathjs.all)
+  if (imported !== undefined) {
+    math.import(imported, {})
+  }
```

Comment:
is it problematic to import multiple times? In that case, could we check if the expressions has already been imported?

Reply:
Yes it is, at first I tried to create one global instance and to call import inside the getSymbols function but it fails because of dupplication.